BCStudentSoftwareDevTeam/celts
# Changes Current Term in Admin Management

This is a PR for Issue #129 . I forked off branch admin_management which is on PR #146 .

This issue is regarding adding a way to set the current term. It is supposed to be on the admin management page. 

I created a jinja for loop that adds a button for each term in the database. When clicked on, the JS in userManagement.js makes the clicked-on button active and the rest inactive. I used an Ajax call to send data from a hidden tag to the userManagement.py controller file. It calls changeCurrentTerm(term) in the userManagement.py logic file. changeCurrentTerm(term) updates the g.current_term's isCurrentTerm key to false and get_by_id(term)'s isCurrentTerm to true. If successful, the page reloads and a flasher appears.

TO TEST:
Go to YOUR_IP_ADDRESS/userManagement
Select a different term. 
Hit "Make Current Term"
The current term should change to the term you chose. 

Additional tests are in test_userManagement.py

NOTE: I asked Alex, and he okayed the implementation of buttons as opposed to a dropdown, but said to make note of it for the future.
